cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,95 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
```

Comment:
The newly-added exception has already been added to this PR (see 91e28c0e9ad8ddbd8bd022883f832cb26b63fc9f). 

Though this PR was designed as a straightforward translation of the existing VCL behaviors (which included the extension-based stripping as listed here), I agree that the broader discussion around a simpler set of cache-behavior path conventions is worth pursuing in follow-on PRs (which this simplified cache-configuration will hopefully make more feasible).